src/auth/__mocks__/auth.controller.ts

```diff
@@ -0,0 +1,17 @@
+import { faker } from '@faker-js/faker';
+
+export class mockAuthController {
+  public signIn(_request: { email: string; password: string }) {
```

Comment:
I think you should be as close to the original method as possible:
```suggestion
  signIn(signInRequest: SignInRequest): Promise<string> {
```

I know I suggested `public` in slack conversation but I think you can drop it here if in the original method youd on't have it either